packages/mantine/src/components/table/__tests__/TablePerPage.spec.tsx

```diff
@@ -109,4 +109,29 @@ describe('Table.PerPage', () => {
             expect.objectContaining({pagination: expect.objectContaining({pageIndex: 0})})
         );
     });
+
+    it('triggers the onChangePage Callback with the right parameters', async () => {
+        const user = userEvent.setup({delay: null});
```

Comment:
Can you avoid the `delay: null` it sometimes cause a lot of issues

Reply:
I tried but the test fails. All the other tests in the suite has it too.  I will check what I can do nonetheless

Reply:
removed